OpenEdXMobile/res/layout/fragment_course_unit_grade.xml

```diff
@@ -5,54 +5,123 @@
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:background="@color/white"
-        android:gravity="center"
         android:orientation="vertical">
 
-        <org.edx.mobile.view.custom.IconImageViewXml
-            android:id="@+id/content_error_icon"
-            android:layout_width="@dimen/unit_unavailable_error_icon_size"
-            android:layout_height="@dimen/unit_unavailable_error_icon_size"
-            android:layout_margin="@dimen/widget_margin_double"
-            app:iconColor="@color/neutralXXDark"
-            app:iconName="fa-laptop" />
-
-        <TextView
-            android:id="@+id/not_available_message"
+        <LinearLayout
```

Comment:
[Suggestion] Wouldn't this layout be more performant + easy to implement if we use ConstraintLayout instead?

Reply:
updated